sample_code/ep359/rev02/t.py

```diff
@@ -0,0 +1,27 @@
+
+def get_thing(): ...
+
+
+def log_thing(e): ...
```

Comment:
Interesting how `flake8` doesn't complain for the unused `e` variable here.

Reply:
unused parameters aren't reported